app/views/claims/pages/index.html.erb

```diff
@@ -1,28 +1,51 @@
+  )) %>
+<% if @school %>
+    <div class="govuk-grid-row">
+    <br><br>
+      <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l"><%= t("organisation_details") %></h1>
+      <%= govuk_summary_list do |summary_list|
+        summary_list.with_row do |row|
+          row.with_key(text: t("organisation_name"))
+          row.with_value(text: @school.name)
+        end; summary_list.with_row do |row|
+          row.with_key(text: t("uk_provider_reference_number"))
+          row.with_value(text: @school.ukprn)
+        end; summary_list.with_row do |row|
+               row.with_key(text: t("unique_reference_number"))
+               row.with_value(text: @school.urn)
+             end; end %>
 
-    <p class="govuk-body">
-      Your application is ready - so long as this page rendered without any errors you're good to go.
-    </p>
-
-    <%= govuk_summary_list(
-      rows: [
-        { key: { text: "Service" }, value: { text: "Claims" } },
-        { key: { text: "Rails version" }, value: { text: Rails.version } },
-        { key: { text: "Ruby version" }, value: { text: RUBY_VERSION } },
-        {
-          key: {
-            text: "GOV.UK Frontend",
-          },
-          value: {
-            text: JSON
-              .parse(File.read(Rails.root.join("package.json")))
-              .dig("dependencies", "govuk-frontend")
-              .tr("^", ""),
-          },
-        },
-      ],
-    ) %>
-  </div>
-</div>
+        <h2 class="govuk-heading-m govuk-!-margin-top-9">Contact details</h2>
+        <%= govuk_summary_list do |summary_list|
+          summary_list.with_row do |row|
+          row.with_key(text: t("email_address"))
+          row.with_value do %>
+            <a href="contact@example.com">contact@example.com</a>
+          <% end;row.with_action(text: t('change'), visually_hidden_text:  t("email_address"), href: '/contact-details')
+          end;summary_list.with_row do |row|
+            row.with_key(text: 'Telephone number')
+            row.with_value(text: @school.telephone)
+            row.with_action(text: t('change'), visually_hidden_text:  t("email_address"), href: '/contact-details')
+          end;summary_list.with_row do |row|
+              row.with_key(text: 'Website')
+              row.with_value do %>
+                <a href="<%= @school.website %>"><%= @school.website %></a>
+              <% end;row.with_action(text: t('change'), visually_hidden_text:  t('email_address'), href: '/contact-details')
+            end;summary_list.with_row do |row|
+              row.with_key(text: t('address'))
+              row.with_value do %>
+                <p class="govuk-body">
+                  <%= @school.address1 %><br>
+                  <%= @school.address2 %><br>
+                  <%= @school.address3 %><br>
+              </p>
+              <% end;row.with_action(text:  t('change'), visually_hidden_text:  t('email_address'), href: '/contact-details')
```

Comment:
Hmm, are we sure this is linted correctly? Many lines start with `end;`.

Reply:
I am finding this difficult to read -- not sure why the linter allowed this. And sometimes we have `end;row` and sometimes we have `end; row`.